### PR TITLE
compatibility with mantisbt 1.3

### PR DESCRIPTION
**Pull request by cproensa**
manages required mantis core version, and jqueryui plugin
change inline css to properly hide the form on page load
removes closing php tag

### CODE REVIEW

File: InlineColumnConfiguration/InlineColumnConfiguration.php [hunk 1]
```diff
@@ -22,10 +22,23 @@ public function register() {
...
+			$this->requires = array(
+				"MantisCore" => "1.3"
+			);
+		}
+		else {
+			$this->requires = array(
+				"MantisCore" => "1.2.6",
+				"jQueryUI" => "1.8"
+			);
+		}
```
Comment (dregad) [inline InlineColumnConfiguration/InlineColumnConfiguration.php:37]: Considering the differences between the 2 versions, I'm personally not a big fan of maintaining compatibility like this (I'd rather have 2 distinct branches)

That being said, if it has to be done this way, why not simply use 

```
if( version_compare( MANTIS_VERSION, '1.3', '<' ) ) {
    # 1.2.x
} else {
    # 1.3.x or higher
}
```

**Comment by cproensa** [comment]
Thanks for the tip
I borrowed that code snippet from the plugin_api version check... your suggestion is cleaner

For the state of this plugin, the only modifications needed for v1.3 is remove the jQueryUI dependency, and the quirk about the inline css (seems like inline css does not work in 1.3 anymore)

i will revisit these commits with the clean-up

**Comment by dregad** [comment]
> seems like inline css does not work in 1.3 anymore

Most likely due to CSP.

**Comment by cproensa** [comment]
the version compare is rewritten
please ignore 3rd commit when reviewing, which is only cleaning spaces (but messes the diffs)

**Comment by dregad** [comment]
@rombert this looks good to me, tested the behavior both with 1.2 and 1.3, everything seems to be working fine. Let me know if I can merge this or if you want to handle it. 

Also, there is no corresponding issue for it in the [tracker](http://www.mantisbt.org/bugs/search.php?project_id=14&sticky_issues=1&sortby=last_updated&dir=DESC), maybe you want one added.

**Comment by dregad** [comment]
@cproensa maybe the code cleanup commit could be moved to a distinct PR, to facilitate review and avoid mixing unrelated changes with your fix.

**Comment by cproensa** [comment]
dregad, already removed the 3rd commit, would apply it later then

**Comment by dregad** [comment]
Cheers.

**Comment by rombert** [comment]
@dregad - feel free to merge if everything looks fine to you. As for an issue in the tracker, I don't think it's mandatory as I don't think I've cut an official release yet ( but maybe after merging the current PRs is a good time )

**Pull request closed without merging**